### PR TITLE
Updated for OpenCart 3.x - iFrame

### PR DESCRIPTION
**Pull request by octaviandragan**
Updated files to match the newer versions of OpenCart - iFrame flow

### CODE REVIEW

File: catalog/view/theme/default/template/extension/payment/twocheckout.twig [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  <div class="buttons">
+    <div class="pull-right">
+      <input type="submit" value="{{ button_confirm }}" class="btn btn-primary" />
+    </div>
+  </div>
+</form>
+{% if display %}
+<script type="text/javascript" src="https://www.2checkout.com/static/checkout/javascript/direct.min.js"></script>
+{% endif %}
+test
```
Comment (craigchristenson) [inline catalog/view/theme/default/template/extension/payment/twocheckout.twig:41]: left in test statement

File: README.md [hunk 2]
```diff
@@ -1,33 +1,77 @@
...
-
-1. Sign in to your 2Checkout account. 
-2. Click the **Account** tab and **Site Management** subcategory. 
-3. Under **Direct Return** select **Header Redirect** or **Given links back to my website**.
-4. Enter your **Secret Word**._(Must be the same value entered in your OpenCart admin.)_
-5. Set the **Approved URL** to http://www.yourstore.com/index.php?route=payment/twocheckout/callback _(Replace http://www.yourstore.com with the actual URL to your store.)_
-6. Click **Save Changes**. 
-
-Please feel free to contact 2Checkout directly with any integration questions.
+# OpenCart
```
Comment (craigchristenson) [inline README.md:1]: Copied opencarts readme over the extensions

Comment (craigchristenson) [reply README.md:1]: Would be a good idea to update the readme to insure that the instructions are correct and that it now supports opencart 3 only.

**Review by craigchristenson — CHANGES_REQUESTED** [review]

File: catalog/view/theme/default/template/extension/payment/twocheckout.twig (hunk 1 above)
Comment (octaviandragan) [reply catalog/view/theme/default/template/extension/payment/twocheckout.twig:41]: Removed the "test" line.

File: README.md (hunk 2 above)
Comment (octaviandragan) [reply README.md:1]: Updated the Readme.md file

**Review by craigchristenson — APPROVED** [review]
Awesome thanks man

**Pull request merged**